### ISSUE TITLE
packer-azure build hangs while deleting the temp resource groups for centos

### ISSUE THREAD

**Issue by deepak-shrivastava** [Azure/packer-azure#255]
Hi,
I am trying to provision my packer build (builder azure-arm) but the packer build hangs once the provisioning is complete. It seems like packer is unable to delete the temp resource group that it created. I am making use of centos image. Can you please help???

My provisioner doesnt have anything special. for testing im just creating a folder and coping a file to it. 
I just keep getting the logs message as:

**azure-arm: Cleanup requested, deleting resource group..**.

I tried adding the following block in my provisioner as well:

{
"inline":[
"sudo /usr/sbin/waagent -force -deprovision+user && export HISTSIZE=0 && sync"
],
"inline_shebang": "/bin/sh -x",
"type": "shell",
"skip_clean": true
}

Note: i am making use of the latest packer version v0.10.1. My work is blocked coz of this.

**Comment by boumenot** [comment]
Support for Azure has been folded into the official Packer [repository](http://github.com/mitchellh/Packer).  The version v0.10.1 is sourced from that repository.

There was an [issue][1] that has been fixed, but not released to address your issue (I think).  I would need more details to confirm, but you can review the details in the existing [issue][1].

There are [examples][2] in the official repository for using Azure and Packer.

I'm closing this issue because you're referencing code that does not exist in this repository.  Please re-open the issue if necessary in the official repository.

[1]: https://github.com/mitchellh/packer/issues/3515
[2]: https://github.com/mitchellh/packer/tree/master/examples/azure
